rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinUtilsTest.java

```diff
@@ -224,6 +224,40 @@ void getFirstInputSigHash_forP2shP2wshTx_shouldReturnExpectedLegacyTxSigHash() {
         assertEquals(expectedSigHash, sigHash.get());
     }
 
+    @Test
+    void extractRedeemScriptFromInput_forP2shP2wshTx_withDifferentRedeemScriptInputs_shouldExtractThemProperly() {
```

Comment:
```suggestion
    void extractRedeemScriptFromInput_forTxThatHasP2shAndP2shP2wshInputs_shouldExtractThemProperly() {
```

wdyt? a bit more accurate

Reply:
done!